src/pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,22 @@
+import { Navbar } from "../../components/Navbar/Navbar"
+import Timer from "../../assets/technique.png"
+import "./Homepage.css"
+
+export const Homepage = () => {
+    return (
+        <div className="page-container">
+            <Navbar />
+            <main className="page-content homepage-content">
+                <div className="homepage-text">
+                    <div className="heading-1">An easy to use pomodoro for an effective human.</div>
+                    <div className="homepage-links">
+                        <a className="button" href="http://https://medium.com/keep-productive/how-to-use-a-pomodoro-timer-f9d172132981" target="_blank" rel="noopener noreferrer">What is Pomodoro technique?</a>
```

Comment:
The `href` attribute value to medium article is wrong.

Reply:
have fixed it, thank you for pointing out